Update index.md (#26966)
mdn/content: files/en-us/web/html/element/th/index.md

    - `row`: The header relates to all cells of the row it belongs to.
    - `col`: The header relates to all cells of the column it belongs to.

>     - `rowgroup`: The row group state means the header cell applies to all the remaining cells in the row group.

Reviewer: Can we just say:

```suggestion
    - `rowgroup`: The header belongs to a rowgroup and relates to all of its cells.
```

...to be consistent with the others? @andypillip , does this address the issue as you see it?
